### ISSUE TITLE
fugitive#detect() removed

### ISSUE THREAD

**Issue by Geraint** [idanarye/vim-merginal#42]
I'm getting an error when I try to open Merginal:

```
Error detected while processing function merginal#openMerginalBuffer[5]..31[27]..fugitive#detect:
line    1:
E605: Exception not caught: Third party code is using fugitive#detect() which has been removed. Contact the author if you have a reason to still use it
Error detected while processing function merginal#openMerginalBuffer[5]..31:
line   27:
E171: Missing :endif
```

I've tried removing all plugins except Merginal and Fugitive, but the problem persists.

I had a quick look at the Fugitive repo, and a recent change may have broken Merginal: https://github.com/tpope/vim-fugitive/commit/cd7db1d57cc991b27d9c9ce6552faea3075ada61#diff-4c8459efc2f0fea43a9d26047a9e5e06R6328

**Comment by idanarye** [comment]
Yes, I fixed that in develop: https://github.com/idanarye/vim-merginal/commit/cf9859e45e402620503fb2c4090bc623d5e8e5a8

I'll do a release later today

**Comment by idanarye** [comment]
Released `2.1.2`, which has the fix.
